frontend_app_admin_workspace_user/src/container/AdminWorkspaceUser.jsx

```diff
@@ -219,25 +218,21 @@ export class AdminWorkspaceUser extends React.Component {
 
     const breadcrumbsList = []
 
-    let page = null
+    let pageUrl = null
     let label = null
     if (state.config.type === 'workspace') {
-      page = PAGE.ADMIN.WORKSPACE
+      pageUrl = PAGE.ADMIN.WORKSPACE
       label = props.t('Space management')
     } else if (state.config.type === 'user') {
-      page = PAGE.ADMIN.USER
+      pageUrl = PAGE.ADMIN.USER
       label = props.t('User account management')
     }
     breadcrumbsList.push({
-      link: <Link to={page}>{label}</Link>,
+      link: pageUrl,
       type: BREADCRUMBS_TYPE.CORE,
       label: label
     })
```

Comment:
`isALink: true`  is not necessary in this case?

Reply:
is should yes, will fix

Reply:
fixed